Source/FolioReaderKit.swift

```diff
@@ -127,9 +130,22 @@ open class FolioReader: NSObject {
     fileprivate var defaults: FolioReaderUserDefaults {
         return FolioReaderUserDefaults(withIdentifier: self.readerContainer?.readerConfig.identifier)
     }
+
+    // Add necessary observers
+    fileprivate func addObservers() {
+        removeObservers()
+        NotificationCenter.default.addObserver(self, selector: #selector(saveReaderState), name: .UIApplicationWillResignActive, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(saveReaderState), name: .UIApplicationWillTerminate, object: nil)
```

Comment:
It should because each observer is added to `self` and it is a different object in memory.